src/Umbraco.Core/Persistence/UmbracoDatabase.cs

```diff
@@ -88,6 +90,22 @@ public override DbCommand CreateCommand(DbConnection connection, CommandType com
             return command;
         }
 
+        public void AddParameter(DbCommand cmd, PocoColumn column, object value)
+        {
+            if (DatabaseType.IsSqlServer())
+            {
+                if (column.ColumnType == typeof(byte[]) && value == null)
+                {
+                    value = new SqlParameter("_bytes", SqlDbType.VarBinary, -1)
+                    {
+                        Value = DBNull.Value
+                    };
+                }
+            }
+
+            base.AddParameter(cmd, value);
+        }
```

Comment:
Looks good, the same solution as in NPoco https://github.com/schotime/NPoco/issues/142 
I wonder why it's not going through that code though, we're using NPoco 3.9 and this was released in NPoco v3.1 🤔

Not that it matters much, I'm happy to put a workaround in if we need it, just curious!